basic/src/test/java/org/jboss/hal/testsuite/test/configuration/io/BufferPoolTestCase.java

```diff
@@ -0,0 +1,178 @@
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+@RunWith(Arquillian.class)
+@RunAsClient
+public class BufferPoolTestCase {
+
+    @Drone
+    private WebDriver browser;
+
+    @Page
+    private IOSubsystemPage page;
+
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final Administration administration = new Administration(client);
+    private static final Operations operations = new Operations(client);
+
+    private static final String
+            BUFFERS_PER_SLICE = "buffers-per-slice",
+            BUFFER_SIZE = "buffer-size",
+            DIRECT_BUFFERS = "direct-buffers";
+
+    private static final Address
+            IO_SUBSYSTEM_ADDRESS = Address.subsystem("io"),
+            BUFFER_POOL_TBA_ADDRESS = IO_SUBSYSTEM_ADDRESS.and("buffer-pool", "buffer-pool-tba" + RandomStringUtils.randomAlphanumeric(7)),
+            BUFFER_POOL_TBR_ADDRESS = IO_SUBSYSTEM_ADDRESS.and("buffer-pool", "buffer-pool-tbr" + RandomStringUtils.randomAlphanumeric(7)),
+            BUFFER_POOL_ADDRESS = IO_SUBSYSTEM_ADDRESS.and("buffer-pool", "buffer-pool" + RandomStringUtils.randomAlphanumeric(7));
+
+    @BeforeClass
+    public static void beforeClass() throws IOException, TimeoutException, InterruptedException {
+        operations.add(BUFFER_POOL_ADDRESS).assertSuccess();
+        operations.add(BUFFER_POOL_TBR_ADDRESS).assertSuccess();
+        administration.reloadIfRequired();
+    }
+
+    @Before
+    public void before() {
+        page.navigateToBufferPools();
+        page.getResourceManager().selectByName(BUFFER_POOL_ADDRESS.getLastPairValue());
+    }
+
+    @AfterClass
+    public static void afterClass() throws IOException, OperationException, TimeoutException, InterruptedException {
+        try {
+            operations.removeIfExists(BUFFER_POOL_ADDRESS);
+            operations.removeIfExists(BUFFER_POOL_TBA_ADDRESS);
+            operations.removeIfExists(BUFFER_POOL_TBR_ADDRESS);
+        } finally {
+            administration.reloadIfRequired();
+        }
+    }
+
+    @Test
+    public void editBuffersPerSlice() throws Exception {
+        final int value = 10;
+        ModelNodeResult modelNodeResult = operations.readAttribute(BUFFER_POOL_ADDRESS, BUFFERS_PER_SLICE);
```

Comment:
Please rename to something like `originalBuffersPerSliceResult` or at least `originalResult`.